pages/api/delegates/index.ts

```diff
@@ -7,25 +7,77 @@ SPDX-License-Identifier: AGPL-3.0-or-later
 */
```

Comment:
There's a merge conflict in this file because Rafa added swagger api definitions to this function, so they should be moved to the legacy endpoint. And it would be nice to add swagger api definitions to this endpoint too so that people can see this endpoint on the api-docs page